### ISSUE TITLE
Problems installing in lektor 3.1.2

### ISSUE THREAD

**Issue by mar10** [lektor/lektor-markdown-admonition#2]
Hi,

I tried to install this plugin and received:
```
$ lektor plugins add markdown-admonition
Package lektor-markdown-admonition (0.3) was added to the project
$ lektor server
Updating packages in /Users/martin/Library/Caches/Lektor/packages/671198c1ca6cd6b0b9d34d67cb943e3d for project
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting lektor-markdown-admonition==0.3
  Could not find a version that satisfies the requirement lektor-markdown-admonition==0.3 (from versions: 0.1)
No matching distribution found for lektor-markdown-admonition==0.3
Traceback (most recent call last):
  File "/usr/local/bin/lektor", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/lektor/cli.py", line 627, in main
    cli.main(args=args, prog_name=name)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/lektor/cli.py", line 380, in server_cmd
    ctx.load_plugins()
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/lektor/cli.py", line 130, in load_plugins
    load_packages(self.get_env(), reinstall=reinstall)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/lektor/packages.py", line 306, in load_packages
    refresh=reinstall)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/lektor/packages.py", line 276, in update_cache
    download_and_install_package(package_root, package, version)
  File "/usr/local/lib/lektor/lib/python2.7/site-packages/lektor/packages.py", line 105, in download_and_install_package
    raise RuntimeError('Failed to install dependency package.')
RuntimeError: Failed to install dependency package.
$ 
```

My project file is now:
```
[project]
name = wwWendt
themes = lektor-theme-simple-strap

[packages]
lektor-markdown-admonition = 0.3
```

**Comment by mar10** [comment]
Setting the version explicitly to 0.1 seems to be a workaround:
```
[packages]
lektor-markdown-admonition = 0.1
```
I noticed that the generated markup
```html
<div class="admonition admonition-info"><p>Test</p></div>
```
is not compatible with my bootstrap theme anyways, so I will disable it for now.
Not sure if I did s.th. wrong, feel free to close this issue.

**Comment by nixjdm** [comment]
The last release of this plugin mistakenly didn't use a universal wheel, and only supports python 3. I will release a fixed version tomorrow. Sorry for the trouble, and thanks for the report!

Also, there may be something in your specific project like a colliding class name (which could be fixed anyway), but in general this should be fine with bootstrap.

**Comment by mar10** [comment]
No worries, using bootstrap markup is an easy and flexible workaround that works in markdown as well:

```html
<div class="alert alert-warning" role="alert">
    <h4 class="alert-heading">
        <i class="fa fa-info-circle"></i>
        Disclaimer
    </h4>
    <p>
        This software is freeware only if used in a non-commercial context. If 
        you wish to use it in a commercial context please 
        <a class="alert-link" href="mailto:freeware@example.com">contact me</a>.
    </p>
    <p class="mb-0">
        This software is beta:
        Use it <strong>AT YOUR OWN RISK</strong>, or don't use it at all!
        I may not be held responsible for any damage it may cause, directly or 
        indirectly.
    </p>
</div>
```
I don't know if it's even worth it to make a plugin support this...

Anyway thanks to you and the team for woking on lektor. Especially themes are a big step forward imo.

**Comment by nixjdm** [comment]
I just released v0.3.1 of this plugin, and 3.1.3 of Lektor for that matter (same issue).

I'm glad you like themes! If you have an issue with or change to simple strap, I'm sure @Andrew-Shay would be happy to see an issue/pr/fork

Thanks for the report!
